EXAMPLE.md

```diff
@@ -0,0 +1,82 @@
+
+---
+RFC: <PR number>
+Status: <Proposed>
+---
+
+# Title
+
+## Abstract
+
+## Motivation
+
+## Design considerations
+
+## Specification
+
+### Commands
+
+(if any)
+1. Command name
+**Request**
+**Response**
+
+### ACL
+
+(if any)
+
+### Append only file
+
+(if any)
+
+### Configuration
+
+(if any)
+
+### Keyspace notifications
+
+(if any)
+
+### Cluster mode 
+
+(if any)
+
+### Module API(s)
+
+(if any)
+
+### RDB
+
+(if any)
+
+### Replication
+
+(if any)
+
+### Networking
+
+(if any)
```

Comment:
What does networking mean?

Reply:
Client/protocol changes.

Reply:
Maybe we should add a one sentence explanation of each of these sections. Might also address Ping's concern below?